examples/spark2/src/main/java/org/apache/carbondata/examples/sdk/CarbonReaderExample.java

```diff
@@ -44,11 +44,10 @@ public static void main(String[] args) {
             fields[1] = new Field("age", DataTypes.INT);
 
             CarbonWriter writer = CarbonWriter.builder()
-                    .withSchema(new Schema(fields))
```

Comment:
Is withSchema is optional earlier or mandatory to provide ?

Reply:
mandatory to provide